### ISSUE TITLE
Dual-logo targeting on dual-branded sites

### ISSUE THREAD

**Issue by tomstringer** [marcom-unimelb/unimelb-design-system#639]
Requested by Melbourne School of Government.

#### Expected behaviour
There are a number of sites that use dual branded logos, approved by the Branding team and implemented using a single svg file (see electionwatch.unimelb.edu.au and vca.unimelb.edu.au for examples).

This request is to make the two logos separately targetable, so a click on the University logo takes you to the University homepage, but a click on the School of Government logo takes you to the homepage of that school.

#### Actual behaviour
Currently these dual logos are served with a single svg file and a single targeted link. It's only possible to make this dual logo go to a single location. In most cases (as above) this is the uni homepage, but in some instances, it's the site homepage (see http://ihouse.unimelb.edu.au).

#### Steps to reproduce
See screenshots of MSOG's Election Watch branding.
![current behaviour](https://cloud.githubusercontent.com/assets/924501/15597382/c322e4bc-2412-11e6-8593-ac04a63d033e.png)

![desired](https://cloud.githubusercontent.com/assets/924501/15597381/c2d9afc2-2412-11e6-8449-1b84f98c410d.png)

**Comment by waitingallday** [comment]
I think this is right, but I'm not sure if we should change the behaviour now.. @deanbrobinson @herrweis ?

**Comment by herrweis** [comment]
Yes, this was chosen deliberately when we implemented the VCA design. it is really confusing for the user to go to two different locations — that's why the faculty/school is the first position (including a home icon) in the breadcrumb. This problem will probably be addressed in the brand architecture project as well, so we should not implement something new that might change again in 6 months...

**Comment by tomstringer** [comment]
No worries. Happy to feed that back.

**Comment by deanbrobinson** [comment]
For what it's worth I think the expected behaviour sounds logical. It's crossed my mind before. Hopefully the branding architecture project will sort it out.

**Comment by K-bhatia** [comment]
Graduate Online website would also like to request the logo URL for our website to change to online.unimelb.edu.au. Currently when prospective student go to one of our landing pages like -online.unimelb.edu.au/lp/ageing-a, upon clicking the logo he is directed to the main Uni website. This is a major customer experience issue, which needs to be addressed.
